### ISSUE TITLE
🐛  (usability/accessibility) Links not visible

### ISSUE THREAD

**Issue by programmiri** [Aiven-Open/klaw-docs#89]
# Description

This text contains a text link. 

<img width="262" alt="Screenshot 2023-09-22 at 17 06 30" src="https://github.com/Aiven-Open/klaw-docs/assets/943800/95c1b47f-813a-4329-aaf3-92bc3d8b267f">

Links must be identifiable for users, by color but also non-Color indicators.
They need a hover and focus state. 

## Todo
update styles
